.github/ISSUE_TEMPLATE_tmpl.md

```diff
@@ -1,16 +1,16 @@
 ## Please follow the guide below
 
-- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
-- Put an `x` into all the boxes [ ] relevant to your *issue* (like that [x])
-- Use *Preview* tab to see how your issue will actually look like
+- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly.
+- Put an `x` into all the boxes [ ] relevant to your *issue* (like this: `[x]`).
+- Use the *Preview* tab to see what your issue will actually look like.
```

Comment:
What's the point of trailing dots? They are not used in the rest of template for lists' items.